lib/9T/ext4mag.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
```

Comment:
is that different from the other ext4mag scripts in the repo?

Reply:
yes, but need it there to get things working.